applications/serial_lte_modem/doc/TCPIP_AT_commands.rst

```diff
@@ -229,16 +225,177 @@ Response syntax
 
 * The ``<sec_tag>`` parameter is an integer.
   It indicates to the modem the credential of the security tag used for establishing a secure connection.
+  It is associated with the certificate or PSK.
+  Specifying the ``<sec_tag>`` is mandatory when opening a socket.
+  When TLS/DTLS is expected, the credentials should be stored on the modem side by ``AT%XCMNG`` or by the Nordic nRF Connect/LTE Link Monitor tool.
+  The modem needs to be in the offline state.
+  The DTLS server is not supported.
+
+* The ``<peer_verify>`` parameter can accept one of the following values:
+
+  * ``0`` - None (default for server role)
+  * ``1`` - Optional
+  * ``2`` - Required (default for client role)
+
+* The ``<hostname_verify>`` parameter can accept one of the following values:
+
+  * ``0`` - None
+  * ``1`` - Required
+
+  The ``<hostname_verify>`` parameter only apply to client role.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XSSOCKET: <handle>[,<type>,<protocol>]
```

Comment:
> Normal socket and secure socket share the same URC so no need of change

If so, this doesn't seem right then

Reply:
yes this is not right, should be ` #XSOCKET: <handle>[,<type>,<protocol>]`.
It's so in the code but not so in the doc